### PR TITLE
Update account not registered error with hint

### PR DESCRIPTION
**Pull request by maaktweluit**
To help users resolve this error themselves: Add a hint that solves this problem in 99% of the cases.

### CODE REVIEW

File: core/payment/src/error.rs [hunk 1]
```diff
@@ -112,7 +112,7 @@ pub mod processor {
     use ya_core_model::payment::public::SendError;
 
     #[derive(thiserror::Error, Debug)]
-    #[error("Account not registered. platform={platform} address={address} mode={mode:?}")]
+    #[error("Account not registered. Hint: Did you run `yagna payment init -r` after restarting? platform={platform} address={address} mode={mode:?}")]
```
Comment (Wiezzel) [inline core/payment/src/error.rs:115]: The flag in the error message should depend on `mode` field. In case of receiving payments it's `-p`.

Comment (maaktweluit) [reply core/payment/src/error.rs:115]: i thought this was not needed since -p is called by default

Comment (Wiezzel) [reply core/payment/src/error.rs:115]: Well, it's called by default but only for the default identity. It is possible to create a new identity and not call init.

Comment (maaktweluit) [reply core/payment/src/error.rs]: good one! added the -r or -p dependent on the mode

**Review by Wiezzel — APPROVED** [review]

**Pull request merged**